setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -23,6 +23,7 @@ folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed"
   file source="$(VisualStudioXamlRulesDir)Compile.FSharp.xaml"
   file source="$(VisualStudioXamlRulesDir)Compile.xaml"
   file source="$(VisualStudioXamlRulesDir)COMReference.xaml"
+  file source="$(VisualStudioXamlRulesDir)ConfigurationGeneralPage.xaml"
```

Comment:
We already have a ConfigurationGeneral.xaml; we may want to give this a name that further differentiates it from that one.

Reply:
The similarity was intentional. Do you think it's confusing? In both CPS and the .NET Project System we have `ConfigurationGeneral` to represent general configured data about the project, not intended for display, which is similar to what this does (only this is a "page" for the UI).